cmd/crc/cmd/start_test.go

```diff
@@ -105,7 +105,7 @@ Log in as user:
 
 Use the 'oc' command line interface:
   $ eval $(crc oc-env)
-  $ oc login https://api.crc.testing:6443
+  $ oc login https://api.crc.testing:6443 --username developer
```

Comment:
The documentation uses:
```
oc login -u developer https://api.crc.testing:6443
```
we should use that here as well.

Reply:
done + commit log